angular.json

```diff
@@ -28,9 +28,13 @@
             ],
             "styles": [
               "node_modules/normalize.css/normalize.css",
-              "src/styles/base.scss"
+              "src/styles/base.scss",
+              "./node_modules/froala-editor/css/froala_editor.pkgd.min.css",
+              "./node_modules/froala-editor/css/froala_style.min.css"
```

Comment:
Did you force recreate the container and see @Sanji515 ?

Asking this because is it only me who is getting the below error:
```bash
evalai_ngx   | ** Angular Live Development Server is listening on 0.0.0.0:4200, open your browser on http://localhost:4200/ **
ngx_django   | /usr/local/lib/python3.6/site-packages/requests/__init__.py:80: RequestsDependencyWarning: urllib3 (1.23) or chardet (3.0.4) doesn't match a supported version!
ngx_django   |   RequestsDependencyWarning)
ngx_django   | Performing system checks...
ngx_django   | 
ngx_django   | System check identified no issues (0 silenced).
ngx_django   | July 06, 2019 - 20:16:49
ngx_django   | Django version 1.11.20, using settings 'settings.dev'
ngx_django   | Starting development server at http://0.0.0.0:8000/
ngx_django   | Quit the server with CONTROL-C.
evalai_ngx   | ✖ ｢wdm｣: Error: ENOENT: no such file or directory, open '/code/node_modules/froala-editor/js/froala_editor.pkgd.min.js'
evalai_ngx   |     at Object.openSync (fs.js:436:3)
evalai_ngx   |     at Object.readFileSync (fs.js:336:35)
evalai_ngx   |     at Storage.provideSync (/code/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:98:13)
evalai_ngx   |     at CachedInputFileSystem.readFileSync (/code/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:259:32)
evalai_ngx   |     at Observable._subscribe (/code/node_modules/@ngtools/webpack/src/webpack-input-host.js:35:51)
evalai_ngx   |     at Observable._trySubscribe (/code/node_modules/@ngtools/webpack/node_modules/rxjs/internal/Observable.js:44:25)
evalai_ngx   |     at Observable.subscribe (/code/node_modules/@ngtools/webpack/node_modules/rxjs/internal/Observable.js:30:22)
evalai_ngx   |     at SyncDelegateHost._doSyncCall (/code/node_modules/@angular-devkit/core/src/virtual-fs/host/sync.js:22:20)
evalai_ngx   |     at SyncDelegateHost.read (/code/node_modules/@angular-devkit/core/src/virtual-fs/host/sync.js:49:21)
evalai_ngx   |     at WebpackCompilerHost.readFileBuffer (/code/node_modules/@ngtools/webpack/src/compiler_host.js:125:44)
evalai_ngx   |     at VirtualFileSystemDecorator.readFile (/code/node_modules/@ngtools/webpack/src/virtual_file_system_decorator.js:39:54)
evalai_ngx   |     at /code/node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/scripts-webpack-plugin.js:69:49
evalai_ngx   |     at new Promise (<anonymous>)
evalai_ngx   |     at /code/node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/scripts-webpack-plugin.js:68:24
evalai_ngx   |     at Array.map (<anonymous>)
evalai_ngx   |     at /code/node_modules/@angular-devkit/build-angular/src/angular-cli-files/plugins/scripts-webpack-plugin.js:67:43
evalai_ngx   | ℹ ｢wdm｣: wait until bundle finished: /
```

Reply:
@galipremsagar have you installed the new dependency which I've added? because this styles comes with that

Reply:
Yes, I did. This error went away. You can ignore this.